app/views/vacancies/search/_filters.html.slim

```diff
@@ -89,12 +89,16 @@ ruby:
   html_attributes: { tabindex: "-1", id: "filters-component" }) do |filters_component|
     - filters_component.with_remove_filter_links do |rb|
       - filter_types.each do |filter_type|
-        - rb.with_group(**filter_type.merge(remove_filter_link: { url_helper: :jobs_path, params: vacancies_search.remove_filter_params }))
-    - filters_component.with_group key: "visa_sponsorship_availability", component: f.govuk_collection_check_boxes(:visa_sponsorship_availability, form.visa_sponsorship_availability_options, :first, :last, small: true, legend: { text: t("jobs.filters.visa_sponsorship_availability.legend") }, hint: nil)
+        - unless @landing_page&.hidden_filters&.include?(filter_type[:key].to_s)
+          - rb.with_group(**filter_type.merge(remove_filter_link: { url_helper: :jobs_path, params: vacancies_search.remove_filter_params }))
+    - unless @landing_page&.hidden_filters&.include?("visa_sponsorship")
+      - filters_component.with_group key: "visa_sponsorship_availability", component: f.govuk_collection_check_boxes(:visa_sponsorship_availability, form.visa_sponsorship_availability_options, :first, :last, small: true, legend: { text: t("jobs.filters.visa_sponsorship_availability.legend") }, hint: nil)
     = render "/vacancies/search/job_roles_filters", filters_component: filters_component, f: f, form: form
     - filters_component.with_group key: "education_phase", component: f.govuk_collection_check_boxes(:phases, form.phase_options, :first, :last, small: true, legend: { text: t("jobs.filters.phases") }, hint: nil)
-    = render "shared/subjects_filter", filters_component: filters_component, f: f
-    - filters_component.with_group key: "ect_statuses", component: f.govuk_collection_check_boxes(:ect_statuses, form.ect_status_options, :first, :last, small: true, legend: { text: t("jobs.filters.ect_suitable") }, hint: nil)
+    - unless @landing_page&.hidden_filters&.include?("subjects")
+      = render "shared/subjects_filter", filters_component: filters_component, f: f
+    - unless @landing_page&.hidden_filters&.include?("ect_statuses")
+      - filters_component.with_group key: "ect_statuses", component: f.govuk_collection_check_boxes(:ect_statuses, form.ect_status_options, :first, :last, small: true, legend: { text: t("jobs.filters.ect_suitable") }, hint: nil)
```

Comment:
hidden_filters cannot be nil, so I don't think we need these safe_navigation operators in the template